### ISSUE TITLE
Delegates Support

### ISSUE THREAD

**Issue by AithaTarun** [tensorflow/tflite-micro#2604]
Hi,
So recently I came across libarmnnDelegate.so which can be used with tensorflow lite.
Does MicroInterpreter of tensorflow lite micro also supports Delegates ?

Thanks

**Comment by rascani** [comment]
That is correct, TFLite-Micro does not support Delegates and there are no current plans to do so.
